<!-- Robot nodes. -->
<launch>
    <arg name="robot_id" default=''/>
    <arg name="robot_type" default=''/>
    <arg name="base_stations" default=''/>
    <arg name="frontier_robots" default=''/>
    <arg name="relay_robots" default=''/>

    <node pkg="gvgexploration" type="robot.py" name="node$(arg robot_id)" >
        <param name="robot_id" value="$(arg robot_id)"/>
        <param name="robot_type" value="$(arg robot_type)"/>
        <param name="base_stations" value="$(arg base_stations)"/>
        <param name="frontier_robots" value="$(arg frontier_robots)"/>
        <param name="relay_robots" value="$(arg relay_robots)"/>

    </node>

    <node pkg="gvgexploration" type="gvg_explore.py" name="node$(arg robot_id)_explore" >
        <param name="robot_id" value="$(arg robot_id)"/>
        <param name="robot_type" value="$(arg robot_type)"/>
        <param name="base_stations" value="$(arg base_stations)"/>
        <param name="frontier_robots" value="$(arg frontier_robots)"/>
        <param name="relay_robots" value="$(arg relay_robots)"/>
    </node>

</launch>
